src/chttpd/src/chttpd_db.erl

```diff
@@ -569,15 +569,23 @@ db_req(#httpd{method='POST',path_parts=[_,<<"_bulk_docs">>], user_ctx=Ctx}=Req,
             send_json(Req, 417, ErrorsJson)
         end;
     false ->
-        case fabric:update_docs(Db, Docs, [replicated_changes|Options]) of
-        {ok, Errors} ->
-            chttpd_stats:incr_writes(length(Docs)),
-            ErrorsJson = lists:map(fun update_doc_result_to_json/1, Errors),
-            send_json(Req, 201, ErrorsJson);
-        {accepted, Errors} ->
-            chttpd_stats:incr_writes(length(Docs)),
-            ErrorsJson = lists:map(fun update_doc_result_to_json/1, Errors),
-            send_json(Req, 202, ErrorsJson)
+        case lists:any(fun(JsonObj) ->
+            Doc = couch_db:doc_from_json_obj_validate(Db, JsonObj),
+            Doc#doc.revs =:= {0, []} end, DocsArray) of
+        true ->
+            throw({bad_request, ?l2b("`new_edits: false` is "
+            "intended for use only by the replicator.")});
```

Comment:
The statement is true but doesn't help the user as much. For all we know they were trying to build their own replicator so it would be nice provide exact feedback on why we failed their request

Reply:
Changed, thanks for the review!